### PR TITLE
Fix errors, improve menus, views, readme

### PR DESCRIPTION
**Pull request by grindtildeath**

### CODE REVIEW

**Comment by grindtildeath** [comment]
@sylvainvh This PR looks good to me now. Maybe ask @yvaucher again for a code review.

File: account_credit_control/README.rst [hunk 1]
```diff
@@ -18,10 +23,14 @@ applied on all partners in each company, under the Accounting tab.
 
 You are able to specify a particular policy for one partner or one invoice.
 
+.. figure:: path/to/local/image.png
+   :alt: alternative description
+   :width: 600 px
```
Comment (yvaucher) [inline account_credit_control/README.rst]: Does it comes from the template ? You probably don't want that here. As we don't have an image.

File: account_credit_control/README.rst [hunk 2]
```diff
@@ -33,12 +42,45 @@ On each generated line, you have many choices:
...
 
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/92/10.0
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/92/issues>`_. In case of trouble, please
```
Comment (yvaucher) [inline account_credit_control/README.rst]: s/92/account-financial-tools

File: account_credit_control/views/account_invoice.xml [hunk 3]
```diff
@@ -17,7 +17,7 @@
                 <page string="Credit Control"
                       groups="account_credit_control.group_account_credit_control_manager,account_credit_control.group_account_credit_control_user,account_credit_control.group_account_credit_control_info">
                     <group>
-                        <field name="credit_policy_id" widget="selection"
+                        <field name="credit_policy_id"
```
Comment (yvaucher) [inline account_credit_control/views/account_invoice.xml:20]: I think the use of the widget was intentional to forbid creation of policy from this view

File: account_credit_control/views/credit_control_line.xml [hunk 4]
```diff
@@ -118,6 +111,12 @@
                     <filter domain='[]'
                             context="{'group_by': 'manually_overridden'}"
                             icon="terp-document-new" string="Manual change"/>
+                    <separator orientation="vertical"/>
+                    <field name="partner_id"/>
+                    <field name="account_id"/>
+                    <field name="invoice_id"/>
+                    <field name="level"/>
+                    <field name="policy_level_id"/>
```
Comment (yvaucher) [inline account_credit_control/views/credit_control_line.xml]: Those are not group_by, the name="group_by" was probably wrong. AFAIK only `filter` with `group_by` in context are actually group by

**Review by yvaucher — CHANGES_REQUESTED** [review]

**Comment by yvaucher** [comment]
@grindtildeath Did a little review here

**Review by sylvainvh — APPROVED** [review]

File: account_credit_control/views/account_invoice.xml (hunk 3 above)
Comment (grindtildeath) [reply account_credit_control/views/account_invoice.xml:20]: As this field is readonly, widget makes the field displayed as simple text. Removing the widget allows the field to be displayed as a relation.

File: account_credit_control/views/credit_control_line.xml (hunk 4 above)
Comment (grindtildeath) [reply account_credit_control/views/credit_control_line.xml]: The fields do appear when searching for a text, so I now grouped them before filters.

**Review by yvaucher — APPROVED** [review]

**Pull request merged**